### PR TITLE
Add link to displaySurface info in Using the Screen Capture API

### PR DESCRIPTION
**Pull request by alvinhui**
> What was wrong/why is this fix needed? (quick summary only)

> MDN URL of the main page changed

> Issue number (if there is an associated issue)

> Anything else that could help us review it

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
Thanks for the PR, @alvinhui !

Do you think we could replace this section with a link to https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#properties_of_shared_screen_tracks ? As it is we need to maintain this information in two places.

**Comment by alvinhui** [comment]
> Thanks for the PR, @alvinhui !
> 
> Do you think we could replace this section with a link to https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#properties_of_shared_screen_tracks ? As it is we need to maintain this information in two places.

Yes, I think is better add a link for "There are three new constraints added ..." and then remove following Properties.

And keep "None of the constraints are applied ..." content is OK.

**Comment by sideshowbarker** [comment]
> > Thanks for the PR, @alvinhui !
> > Do you think we could replace this section with a link to [developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#properties_of_shared_screen_tracks](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#properties_of_shared_screen_tracks) ? As it is we need to maintain this information in two places.
> 
> Yes, I think is better add a link for "There are three new constraints added ..." and then remove following Properties.
> 
> And keep "None of the constraints are applied ..." content is OK.

Sounds good to me. Can you push a follow-up commit to this PR branch, with those changes?

**Comment by alvinhui** [comment]
> > > Thanks for the PR, @alvinhui !
> > > Do you think we could replace this section with a link to [developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#properties_of_shared_screen_tracks](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#properties_of_shared_screen_tracks) ? As it is we need to maintain this information in two places.
> > 
> > 
> > Yes, I think is better add a link for "There are three new constraints added ..." and then remove following Properties.
> > And keep "None of the constraints are applied ..." content is OK.
> 
> Sounds good to me. Can you push a follow-up commit to this PR branch, with those changes?

let me do it later

**Comment by alvinhui** [comment]
just updated, @sideshowbarker

File: files/en-us/web/api/screen_capture_api/using_screen_capture/index.html [hunk 1]
```diff
@@ -82,25 +82,7 @@ <h3 id="Options_and_constraints">Options and constraints</h3>
...
-  <dt><code>motion</code></dt>
-  <dd>The cursor should only be visible while moving, and, at the discretion of the {{Glossary("user agent")}}, for a brief time before after moving. Then the cursor is removed from the stream.</dd>
-  <dt><code>never</code></dt>
-  <dd>The cursor should never be visible in the generated stream.</dd>
- </dl>
- </dd>
- <dt>{{domxref("MediaTrackConstraints.logicalSurface", "logicalSurface")}}</dt>
- <dd>A Boolean which if <code>true</code> indicates that the capture should include offscreen areas of the source, if there are any.</dd>
-</dl>
+<p>There are same new constraints added to <code>MediaTrackConstraints</code> (as well as to {{domxref("MediaTrackSupportedConstraints")}} and {{domxref("MediaTrackSettings")}}) for configuring a screen capture stream:</p> <a href="/en-US/docs/Web/API/MediaTrackConstraints#properties_of_shared_screen_tracks">Properties of shared screen tracks</a>.
```
Comment (sideshowbarker) [inline files/en-us/web/api/screen_capture_api/using_screen_capture/index.html:85]:
```suggestion
<p>See <a href="/en-US/docs/Web/API/MediaTrackConstraints#properties_of_shared_screen_tracks">Properties of shared screen tracks</a> for details about additional constraints for configuring a screen-capture stream that are added to {{domxref("MediaTrackConstraints")}}, {{domxref("MediaTrackSupportedConstraints")}}, and {{domxref("MediaTrackSettings")}}).</p>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**